### PR TITLE
Data release to version 2020_10_08_19_40_25

### PR DESCRIPTION
**Pull request by shifucun**
Landing page golden file changes look good. Please verify other changes.

### CODE REVIEW

File: golden_response/staging/get_landing_page_data/asm.json [hunk 1]
```diff
@@ -611,7 +626,165 @@
       "RetailDrugDistribution_DrugDistribution_Oxycodone": {},
       "UnemploymentRate_Person": {},
       "UnemploymentRate_Person_Female": {},
-      "UnemploymentRate_Person_Male": {}
+      "UnemploymentRate_Person_Male": {},
+      "WorldBank_BM_TRF_PWKR_CD_DT": {},
```
Comment (tjann) [inline golden_response/staging/get_landing_page_data/asm.json:630]: @shifucun any idea why this didn't have WDI data before?

File: golden_response/staging/get_related_locations/county.json [hunk 2]
```diff
@@ -1,41 +1,17 @@
 {
   "Count_Person": {
-    "relatedPlaces": [
```
Comment (tjann) [inline golden_response/staging/get_related_locations/county.json:3]: This is because we switch to cohort?

**Review by tjann — APPROVED** [review]

File: golden_response/staging/get_landing_page_data/asm.json (hunk 1 above)
Comment (shifucun) [reply golden_response/staging/get_landing_page_data/asm.json:630]: This cache flume should have use the updated chart config with these stat vars?

File: golden_response/staging/get_related_locations/county.json (hunk 2 above)
Comment (shifucun) [reply golden_response/staging/get_related_locations/county.json:3]: The related place is dropped from the flume?

Comment (pradh) [reply golden_response/staging/get_related_locations/county.json:3]: I think Flume drops relatedLocations property, but that hasn't rolled out.

Comment (shifucun) [reply golden_response/staging/get_related_locations/county.json:3]: that's interesting.. but this is okay for staging, as I am going to use cohort for related places: https://github.com/datacommonsorg/mixer/pull/329

**Review by beets — APPROVED** [review]

**Pull request merged**